src/renderer/Default.js

```diff
@@ -94,6 +92,11 @@ export default function Default() {
 
               <LibcoreBusyIndicator />
               <DeviceBusyIndicator />
+              <DebugWrapper>
```

Comment:
I think I put the DebugMock outside of OnboardingElse to ensure that the aimations are disabled on the onboard

Reply:
it might be the reason why the onboarding screenshots fail

Reply:
actually when running in spectron, there is global class spectron-run added to the body, so maybe the global styles defined in DebugMock can be but directly in the global.css under .spectron_run { }

Reply:
I see, we could move the whole thing out then since it's absolutely positioned

Reply:
fixed.